### PR TITLE
Forwarder image cleanup and work on the olm-artifacts-template

### PR DESCRIPTION
**Pull request by wshearn**
Decided to only go with hive logs for now, once I have verified everything is working I will create the selectorsyncset.

Going with hive only for now allows us to verify the build pipeline and the flow from the forwarder to splunk.

### CODE REVIEW

File: hack/olm-registry/olm-artifacts-template.yaml [hunk 1]
```diff
@@ -38,4 +38,19 @@ objects:
...
     name: splunk-forwarder-operator
     source: splunk-forwarder-operator-catalog
-    sourceNamespace: splunk-forwarder-operator
\ No newline at end of file
+    sourceNamespace: splunk-forwarder-operator
+
+- apiVersion: splunkforwarder.managed.openshift.io/v1alpha1
+  kind: SplunkForwarder
+  metadata:
+    name: example-splunkforwarder
```
Comment (jharrington22) [inline hack/olm-registry/olm-artifacts-template.yaml]: example? Should we change the name here?

Comment (wshearn) [reply hack/olm-registry/olm-artifacts-template.yaml]: this is what i get for just copying the example instead of rewriting it :) fixed

**Pull request merged**